app/models/manageiq/providers/amazon/block_storage_manager/refresh_parser_inventory_object.rb

```diff
@@ -0,0 +1,64 @@
+class ManageIQ::Providers::Amazon::BlockStorageManager::RefreshParserInventoryObject < ::ManagerRefresh::RefreshParserInventoryObject
+  include ManageIQ::Providers::Amazon::RefreshHelperMethods
+
+  def populate_inventory_collections
+    log_header = "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS name: [#{inventory.ems.name}] id: [#{inventory.ems.id}]"
+
+    $aws_log.info("#{log_header}...}")
+    get_volumes
+    get_snapshots
+    $aws_log.info("#{log_header}...Complete")
+
+    inventory_collections
+  end
+
+  private
+  def get_volumes
```

Comment:
We're using the same parsing code for RefreshParser and RefreshParserInventoryObject.
Wouldn't it be better if we move it into ManageIQ::Providers::Amazon::RefreshHelperMethods ?